lib/spack/spack/cmd/buildcache.py

```diff
@@ -363,6 +363,7 @@ def push_fn(args):
                 elide_list([f"    {spec}: {e.__class__.__name__}: {e}" for spec, e in failed], 5)
             ),
         )
+    return 0
```

Comment:
Can you drop this everywhere?

Reply:
maybe, I figured if it was looking for an return code it should return it. But the default is None which coverts to 0 anyway.